lib/utils/npm-packages-exists.js

```diff
@@ -14,8 +15,22 @@ const resolvePackages = require("./resolve-packages").resolvePackages;
 
 module.exports = function npmPackagesExists(pkg) {
 	let acceptedPackages = [];
+
+	function resolvePackagesIfReady() {
+		if (acceptedPackages.length === pkg.length)
+			return resolvePackages(acceptedPackages);
+	}
+
 	pkg.forEach(addon => {
-		//eslint-disable-next-line
+		// The addon is a path to a local folder; no validation is necessary
+		if (fs.existsSync(addon)) {
+			acceptedPackages.push(addon);
+			resolvePackagesIfReady();
+			return;
+		}
+
+		// The addon is on npm; validate name and existence
+		// eslint-disable-next-line
 		if (addon.length <= 14 || addon.slice(0, 14) !== "webpack-addons") {
```

Comment:
If you're passing the relative path you'll get `../webpack-cli/lib/generators/generator.js isn't a valid name.` and `It should be prefixed with 'webpack-addons', but have different suffix.`. Can you do so it says `../webpack-cli/lib/generators/generator.js isn't a valid name or path.` ? And we shouldn't throw the addons error if it's a path

Reply:
I think the `return` statement on line 28 means you'll never make it to that error code.

Reply:
I got there :(
<img width="561" alt="skjermbilde 2018-03-03 kl 02 44 14" src="https://user-images.githubusercontent.com/16735925/36929098-cea0cb58-1e8c-11e8-91c7-a59c80d3da45.png">

Reply:
Ah OK I see. I will fix it.

Reply:
Sweet!

Reply:
Oh wait—it looks like you ran the command pointing at the generator js file. Maybe I misunderstood the requirements. Do we want to allow people to point the `init` command at a javascript file, an npm package, or both?

Reply:
Ugh, I can't reproduce the error you're seeing. Not sure what the deal is. When I pass a relative path to a generator on my filesystem it resolves it without throwing the prefix error. I'm going to be out of town till next Wednesday, so I won't be able to investigate further till then.

Reply:
both 👍